README.md

```diff
@@ -72,24 +72,3 @@ To build the CLI command run:
 ```sh
 make build
 ```
-### Building the Wrapper
```

Comment:
@msdolbey this is important documentation for anyone needing to build the wrapper for development purposes. It should not be removed.

Reply:
Makes sense